oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/CoreModuleConfig.java

```diff
@@ -182,6 +182,12 @@
     @Setter
     private boolean enableEndpointNameGroupingByOpenapi = true;
 
+    /**
+     * The maximum size in bytes allowed for request headers.
+     * Use -1 to disable it.
+     */
+    private int httpMaxRequestHeaderSize = 8192;
```

Comment:
Is this the default value of Jetty?

Reply:
yes, from what I saw, it configures it in many places like this:
https://github.com/eclipse/jetty.project/blob/jetty-9.4.x/jetty-server/src/main/config/etc/jetty.xml#L65

Reply:
Also, this is the value the oap-server logs show when I hit the issue, so defaulting to this one should cause no changes when upgrading an environment.